controllers/updateservice_controller.go

```diff
@@ -636,8 +659,15 @@ func (r *ReconcileCincinnati) ensureSecret(ctx context.Context, reqLogger logr.L
 		reqLogger.Info("Updating Secret", "Namespace", secret.Namespace, "Name", secret.Name)
 		updated := found.DeepCopy()
 		updated.Data = secret.Data
-		return r.client.Update(ctx, updated)
+		return r.Client.Update(ctx, updated)
 	}
 
 	return nil
 }
+
+/*
+func (r *UpdateServiceReconciler) SetupWithManager(mgr ctrl.Manager) error {
```

Comment:
From [the migration docs][1]:

> Copy the code under `func add(mgr manager.Manager, r reconcile.Reconciler)` to `func SetupWithManager`...

I've done that [in][1] [three][2] [hunks][3], if you want to pattern match.


[1]: https://v1-0-x.sdk.operatorframework.io/docs/building-operators/golang/project_migration_guide/#controllers
[2]: https://github.com/openshift/cincinnati-operator/commit/e17da7ad78be074e1b12bd0e80f842d6b43247c9#diff-20974c612e30710860b130edee1c111491fbdbdc904eaf49378e4661ea6a05c3L34-L106
[3]: https://github.com/openshift/cincinnati-operator/commit/e17da7ad78be074e1b12bd0e80f842d6b43247c9#diff-20974c612e30710860b130edee1c111491fbdbdc904eaf49378e4661ea6a05c3R587-R606
[4]: https://github.com/openshift/cincinnati-operator/commit/e17da7ad78be074e1b12bd0e80f842d6b43247c9#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R85-R98

Reply:
Will investigate why I left it and resolve post PR merge.